server/helpers/parseTxBody.js

```diff
@@ -12,8 +12,14 @@ const parseTxBodyOutAmount = (txBody) => {
    * amount = coins | [coins, multiasset]
    */
   const firstTxOutput = cbor.decode(txBody)[0].get(1)[0]
-  const outputAmount = firstTxOutput ? firstTxOutput[1] : 0
-  return coinsFromOutputAmount(outputAmount)
+  // see https://github.com/input-output-hk/cardano-ledger/blob/master/eras/babbage/test-suite/cddl-files/babbage.cddl#L75
+  const parseValueFromOutput = (out) =>
```

Comment:
The function parameter here does not seem to be used. 
Wouldn't it be sufficient to write this as:
```ts
  const parsedValueFromOutput =
    firstTxOutput instanceof Map
      ? firstTxOutput.get(1) // babbage-era
      : firstTxOutput[1] // pre-babbage

  const outputValue = firstTxOutput ? parsedValueFromOutput : 0
  return coinsFromOutputValue(outputValue)
```

Reply:
fixed :+1: It also puzzled me why didn't eslint warn me about such basic mistake and turns out the `no-unused-vars` eslint rule wasn't configured to check for unused function parameters, so I enabled it

Reply:
addressed in https://github.com/vacuumlabs/adalite/pull/1327/commits/bfb1a66bcc25e55ad1238650d1df6185019dae9f